src/test/java/seedu/duke/stubs/ItemStubs.java

```diff
@@ -0,0 +1,41 @@
+package seedu.duke.stubs;
+
+import seedu.duke.data.Item;
+
+public class ItemStubs {
+
+    public static final String MARKER_NAME = "Markers";
+    public static final int MARKER_QUANTITY = 3;
+    public static final String MARKER_DESCRIPTION = "Drawing";
+    public static final Item ITEM_MARKER = generateItem(MARKER_NAME, MARKER_QUANTITY, MARKER_DESCRIPTION);
```

Comment:
Are we able to directly use Item constructor here instead of creating a static method `generateItem`? 

```suggestion
    public static final Item ITEM_MARKER = new Item(MARKER_NAME, MARKER_QUANTITY, MARKER_DESCRIPTION);
```

Reply:
In the future, if Item() constructor has any changes I think it'd be easier to maintain generateItem instead of new Item() calls every time. Thoughts?

Reply:
I see, okay.